server/export.js

```diff
@@ -73,13 +83,13 @@ export default async function (dir, options, configuration) {
 
   // Get the exportPathMap from the `next.config.js`
   if (typeof nextConfig.exportPathMap !== 'function') {
-    printAndExit(
-      '> Could not find "exportPathMap" function inside "next.config.js"\n' +
-      '> "next export" uses that function to build html pages.'
-    )
+    console.log('> No "exportPathMap" found in "next.config.js". Generating map from "./pages"')
+    nextConfig.exportPathMap = async (defaultMap) => {
```

Comment:
Log a message on this is exporting all the pages or something similar.

Reply:
`Generating map from "./pages"` seems clear to me 🤔 @rauchg wrote it

Reply:
I mean the `Generating map from "./pages".`
It's bit of unclear what's map is. If the user created the `exportPathMap` before it's clear. 
Otherwise, he/she might be looking at what's this `map` is.